js/eiti.js

```diff
@@ -635,17 +635,6 @@
     try {
       var e = new CustomEvent('foo'); // jshint ignore:line
     } catch (error) {
-      function CustomEvent(event, params) {
```

Comment:
@shawnbot is this what you had in mind? I removed this function all together and required the `custom-event` module [here](https://github.com/18F/doi-extractives-data/pull/1217/files#diff-9d87fecdcf0f321286d9d33562860b0bR21)